src/com/pty4j/windows/winpty/WinPty.java

```diff
@@ -107,7 +107,7 @@ public class WinPty {
           Thread.sleep(10);
         }
         catch (InterruptedException e) {
-          e.printStackTrace();
+          LOG.error("Cannot sleep to workaround extra newlines issue", e);
```

Comment:
Let's make it `LOG.warn`? These errors might be more likely to be reproduced on slow system, because of `sleep(10)`. Also, WinPty is outdated now, no issues are fixed there.